src/auth/utils.ts

```diff
@@ -375,7 +375,8 @@ export function createConnectionPrompter(auth: Auth, type?: 'iam' | 'sso') {
         return 2
     }
 
-    const prompter = createQuickPick(loadItems(), {
+    const excludeSso = type === 'iam-only'
```

Comment:
Does 'iam' include sso connections? That seems confusing.

(Off-topic for this PR: maybe `type` should be a list instead of only 1 choice.)

Reply:
Ideally, yes, `sam sync` and App Composer should support SSO. However, running `sam sync` with SSO auth currently fails and gives an ambiguous error message to the user. This is meant to be a quick fix to reduce user confusion as to why their sync isn't working.

Reply:
This quick fix adds a maintenance cost because the code changes are spread out. That isn't easy to reverse, so isn't acceptable as a quick fix.

Reply:
What does the `createConnectionPrompter` menu look like after this change?

Reply:
Ok I get it now: the filter credentials menu will prompt the user during the sam-sync flow, making it easy for the user to "do the right thing".

This makes sense.